modules/apps/object/object-test/src/testFunctional/macros/Picklist.macro

```diff
@@ -293,6 +306,16 @@ definition {
 			userPassword = ${userPassword});
 	}
 
+	macro gotoPicklistExport {
```

Comment:
I've found the macro, but it cannot be used on the picklist, not yet, at least. I've spoken to some guys on my team, and we discovered it is a bug. Once it is fixed, we can use the macro, but I think we have to stay with this new macro for the moment.

[Bug link](https://issues.liferay.com/browse/LPS-177982)

Reply:
I think the macro should work regardless of the bug since you can set "menuItem" = Export or Export as JSON on your testcase level. Once the bug is looked at, the variable can be changed if it affects the test.

Reply:
You can use Picklist.optionsPicklist(); to export

Reply:
Ok! Thanks for the feedback!